lib/i18n/app_de.json

```
@@ -190,6 +190,17 @@
     "profiles": "Mitglieder",
     "label": "Mitglieder",
     "noResult": "Leider gab es einen Fehler beim Laden des Profils.",
+    "profileImage": {
+      "update": "Foto bearbeiten",
```

:thinking: Not sure if `bearbeiten` is actually the right word here. If there is none set, I would expect it to be `Profilbild hinzufügen` and if a picture is set, perhaps ` Profilbild ändern`? But I guess that should be checked first.
```suggestion
      "update": "Profilbild bearbeiten",
```

---

I get your point. But as far there is no further specification, I would keep it like this.
The text is taken from the figma file, so I would keep it also.

---

I changed the wording to "Profilbild bearbeiten".